sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -114,11 +110,9 @@ namespace Azure { namespace Core {
     {
     }
 
-    time_point CancelWhen() const;
-
   public:
     /**
-     * @brief Construct a new context with no expiration, and no value associated.
+     * @brief Construct a new context with no deadline, and no value associated.
```

Comment:
Do we have a concept of `max` or `infinite` deadline? Should we use that rather than `no deadline`?

Reply:
We treat `DateTime::max()` as infinite deadline. Which, for the user, the better way to think of, as "no deadline". `HasDeadline()` method naming corresponds to that.